wallet/api.go

```diff
@@ -396,6 +397,17 @@ func (api *API) GetTxListByAddr(addr string, stype int32, page int32, pageSize i
 	return *rs, err
 }
 
+//TxSign TxSign by privkeyStr
```

Comment:
I doubt the case to export TxSign as one public RPC method and use private-key directly. It's too dangerous to lead the end-user to depend on the API.